frontend/src/components/dashboard/campaigns/announcement-modal/AnnouncementModal.module.scss

```diff
@@ -34,9 +32,21 @@ $title-height: $height-4;
 }
 
 .title {
-  text-align: center;
   margin-bottom: 0px;
+}
+
+.titleTop {
+  margin-top: 0px;
+}
+
+.titleCentered {
+  margin-top: 25px;
```

Comment:
Based on our [design system](https://www.figma.com/file/Wd6jTqpdaqaItVLoyk1Me2/%F0%9F%8E%A8Postman-Design-System), we generally use multiples of 4 for spacing values. 

![image](https://user-images.githubusercontent.com/3666479/108017650-63dc2b00-7050-11eb-9ec5-30b63554f31e.png)

Reply:
Should I change the 25px margins & paddings to 24px instead? 25px was the original margin being used, and I'm not sure if changing it to 32px would be too major a change.

Reply:
Can take reference from the figma design.

Reply:
Oh I didn't realise that I could view the dimensions by signing in 😂

Reply:
Seeing how the margins are somewhat different for the 2 types of modals, I'm thinking that maybe it's better to break out the modal into 2 different components: `GraphicAnnouncementModal` and `VideoAnnouncementModal`. Then, `AnnouncementModal` would render the appropriate component based on the return value of the `isVideoUrl` method

Reply:
I have broken out the announcement modal into `GraphicAnnouncementModal` and `VideoAnnouncementModal` components. At the same time, I've adjusted the intra-modal margins to be the same as the ones in the Figma design.

However, I have not adjusted the top and bottom margins (from the top-most element to the top edge of the modal, and from the bottom-most element to the bottom edge of the modal). This is because those margins are defined in `Modal.module.scss`, which is common for all components built upon `Modal`. If I adjust those margins, I will have to test every single component using `Modal`, which isn't within the scope of this PR.

Reply:
Yep. That's fine.